wiki/en/misc/general.yml

```diff
@@ -9,12 +9,12 @@ nav:
   btnOpenNavigation: "Open navigation"
   altJamulusIcon:    "Jamulus Icon. Links to homepage"
 wiki:
-  titleAdd:          "– Jamulus Wiki"
+  titleAdd:          "– Jamulus Website"
 kb:
   titleAdd:          "– Jamulus Knowledge Base"
   by:                "by"
   kbintro:           "<strong>Community knowledge base</strong><br /> Feel free to add your own article!"
-  mainpage:          "KB overview"
+  mainpage:          "Knolwege Base overview"
```

Comment:
> Knolwege

"Knowledge".

Reply:
I fixed that earlier. How come it isn't appearing? https://github.com/gilgongo/jamuluswebsite/blob/d61a7ca28aedf85ee6c59ce0356118e666829b89/wiki/en/misc/general.yml

Reply:
It does appear here now:
https://github.com/jamulussoftware/jamuluswebsite/pull/721/files

I've only two explanations for this:
- @pljones might have started his review before your commit, but might have submitted past your commit.
- A Github cache coherency bug. It's still strange that this thread does not carry an "Outdated" tag. I think (but I'm no longer sure!) that I saw the typo fix missing when visiting the above URL.

During testing, I visited the following URL. It *might* have cleared some Github cache:
https://github.com/jamulussoftware/jamuluswebsite/compare/next-release..gilgongo:asio-2